Source/DataSources/exportKml.js

```diff
@@ -221,6 +221,33 @@ IdManager.prototype.get = function (id) {
 
   return id.toString() + "-" + ++ids[id];
 };
+/**
+ * @variation 2 KML return
```

Comment:
Changes in this file break exportKml JSDoc output completely.

Reply:
Well this is weird.  The version I'm looking at right now is not what I committed.  I saw that there's some kind of commit hook to run a prettifier -- could that be the culprit?  It's important to have

````js
/**
 * Body of documentation
 *
 * @variation 1
 * @param ...
 * @example (for variation 1)
 *//**
 * @variation 2
 * @param ...
 * @example (for variation 2)
 */
````

Exactly thus, with no spaces between the closing tag for the first block and the opening tag for the second.  That's part of the JSDoc spec, apparently, though ironically documentation *about* the spec is a bit dodgy in my experience.  Is it possible to exempt a section or a whole file from being "massaged" by the prettifier?

Reply:
With the blocks aligned correctly, you get this:

![image](https://user-images.githubusercontent.com/943160/83280314-c0539d80-a1d6-11ea-998a-ff1eca89f50a.png)

Reply:
I think I would rather stick with the union return for now.  This type of decision may affect the API elsewhere and our main goal is keeping the API consistent.  I would just revert the file for now.  Feel free to open a separate PR for further discussion.